### DISCUSSION TITLE
Dịch `on/off policy`

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#164] [Terminology] [answered]
@mlbvn/handson-ml  mn cho ý kiến ạ.

**Comment by minhduc0711** [comment, marked as answer]
> The Q-Learning algorithm is called an **off-policy** algorithm because the policy being
trained is not necessarily the one being executed: in the previous code example, the
policy being executed (the exploration policy) is completely random, while the policy
being trained will always choose the actions with the highest Q-Values. Conversely,
the Policy Gradients algorithm is an **on-policy** algorithm: it explores the world using
the policy being trained.

"(không) dựa trên chính sách" chăng?

**Reply by cuongvng** [reply]
`not necessarily` tức là có thể vẫn "dựa trên" 🤔 🤔

**Reply by minhduc0711** [reply]
Hmm vậy thì off-policy là tập con của on-policy à? Nghe có vẻ k đúng lắm..
Mình xem định nghĩa ở sách Sutton-Barto thì có vẻ phân biệt trắng đen luôn:
![image](https://user-images.githubusercontent.com/34294427/177417243-c4af39c4-d8a5-49c4-bf86-31f6a4dd00ae.png)

**Reply by cuongvng** [reply]
vậy cứ chốt cách dịch trên ạ.
